index.js

```diff
@@ -50,209 +61,223 @@
-    true(node) {
-      if (node.value === '--') {
-        node.value = '—'
-      }
-    },
-    /**
-     * Transform three dahes into an em-dash, and two into an en-dash.
-     *
-     * @type {Method}
-     */
-    oldschool(node) {
-      if (node.value === '---') {
-        node.value = '—'
-      } else if (node.value === '--') {
-        node.value = '–'
+const defaultClosingQuotes = {'"': '”', "'": '’'}
+const defaultOpeningQuotes = {'"': '“', "'": '‘'}
+
+/**
+ * @param {Options} options
+ */
+function createEducators(options) {
+  const closingQuotes = options.closingQuotes
+    ? {'"': options.closingQuotes.double, "'": options.closingQuotes.single}
+    : defaultClosingQuotes
+  const openingQuotes = options.openingQuotes
+    ? {'"': options.openingQuotes.double, "'": options.openingQuotes.single}
+    : defaultOpeningQuotes
+
+  const educators = {
+    dashes: {
+      /**
+       * Transform two dahes into an em-dash.
+       *
+       * @type {Method}
+       */
+      true(node) {
+        if (node.value === '--') {
+          node.value = '—'
+        }
+      },
+      /**
+       * Transform three dahes into an em-dash, and two into an en-dash.
+       *
+       * @type {Method}
+       */
+      oldschool(node) {
+        if (node.value === '---') {
+          node.value = '—'
+        } else if (node.value === '--') {
+          node.value = '–'
+        }
+      },
+      /**
+       * Transform three dahes into an en-dash, and two into an em-dash.
+       *
+       * @type {Method}
+       */
+      inverted(node) {
+        if (node.value === '---') {
+          node.value = '–'
+        } else if (node.value === '--') {
+          node.value = '—'
+        }
       }
     },
-    /**
-     * Transform three dahes into an en-dash, and two into an em-dash.
-     *
-     * @type {Method}
-     */
-    inverted(node) {
-      if (node.value === '---') {
-        node.value = '–'
-      } else if (node.value === '--') {
-        node.value = '—'
-      }
-    }
-  },
-  backticks: {
-    /**
-     * Transform double backticks and single quotes into smart quotes.
-     *
-     * @type {Method}
-     */
-    true(node) {
-      if (node.value === '``') {
-        node.value = '“'
-      } else if (node.value === "''") {
-        node.value = '”'
+    backticks: {
+      /**
+       * Transform double backticks and single quotes into smart quotes.
+       *
+       * @type {Method}
+       */
+      true(node) {
+        if (node.value === '``') {
+          node.value = '“'
+        } else if (node.value === "''") {
+          node.value = '”'
+        }
+      },
+      /**
+       * Transform single and double backticks and single quotes into smart quotes.
+       *
+       * @type {Method}
+       */
+      all(node, index, parent) {
+        educators.backticks.true(node, index, parent)
+
+        if (node.value === '`') {
+          node.value = '‘'
+        } else if (node.value === "'") {
+          node.value = '’'
```

Comment:
Should this also be affected?

Reply:
hmm, good point, yes should probably be affected as well i think

Reply:
On the other hand, that isn’t about `"` (and `'`).
That would perhaps need “primary, secondary” options instead?
Easier to leave it as is for now?

Reply:
oh, ok should i revert https://github.com/retextjs/retext-smartypants/pull/8/commits/59d7be4a4fcc3dd8597b03c3e2e7b4b4e45f37c2 then?